slides.qmd

```diff
@@ -23,10 +23,73 @@ format:
 
 ## Motivation
 
-*Andreas to add content*
+Learning from others: the good, the bad, and the ugly.
+
+Visualisations are encountered everywhere.
+
+
+## Case study 1: Football tickets ##
+
+![](images/Case study-stadium ticket 1.png){fig-alt="Stadium map"}
```

Comment:
Do you want figures centred on the slide?

Reply:
Moved into backup section (i.e., deactivated).